Modelica/Media/Air/ReferenceMoistAir.mo

```diff
@@ -2949,7 +2949,7 @@ for region 2.
       SI.MoleFraction[4] Y
         "Mole fractions of individual components (H2O, N2, O2, Ar) of moist air";
     algorithm
-      if (useDissociation == false) then
+      if (not useDissociation) then
```

Comment:
to me it would be more readable as (and same for the rest):

```
if useDissociation then
  massFraction := {X[1],X[2]*Xi_Air[1],X[2]*Xi_Air[2],X[2]*Xi_Air[3]};
  ...
else
  u:=0;
end if;
```